extensions/optaplanner/runtime/src/main/java/io/quarkus/optaplanner/OptaPlannerBeanProvider.java

```diff
@@ -13,27 +13,18 @@
 
 public class OptaPlannerBeanProvider {
 
-    static volatile SolverConfig solverConfig;
-    static volatile SolverManagerConfig solverManagerConfig;
```

Comment:
@geoand I am very happy to see the static fields disappear :) That was ugly!

Reply:
Yeah, @mkouba has done some great work in improving the creation of bean from runtime values